src/modules/comments/controllers/comments.controller.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Post,
+  Param,
+} from '@nestjs/common';
+
+import { CommentsService } from '../services/comments.service';
+import { Comment } from '../interfaces/comment.interface';
+
+@Controller('comments')
+export class CommentsController {
+  constructor(private readonly commentsService: CommentsService) {}
+
+  @Post('/:post/comments')
```

Comment:
can we name route like this: ```@Post('/:post_id/comments')```, so it would be more clear that we are creating comment that is related to post_id

Reply:
@kolyastepanets this whole controller has prefix route `/posts`, so it's no need to rewrite here `@Post('/posts/:post_id/comments')`.
As for `@Post('/:post_id/comments')` - I agree and will change this